x/pairing/client/cli/tx_simulate_relay_payment.go

```diff
@@ -0,0 +1,211 @@
+				}
+				// Set Sig
+				relaySession.Sig = sig
+
+				relaySessions = append(relaySessions, relaySession)
+			}
+
+			msg := types.NewMsgRelayPayment(
+				clientCtx.GetFromAddress().String(),
+				relaySessions,
+				"",
+			)
+			utils.LavaFormatInfo("Message Data ", utils.Attribute{Key: "msg", Value: msg})
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+	cmd.MarkFlagRequired(flags.FlagFrom)
+	cmd.Flags().Uint64(CuAmountFlag, 0, "CU serviced by the provider")
+	cmd.MarkFlagRequired(CuAmountFlag)
+	cmd.Flags().Uint64(RelayNumFlag, 1, "Number of relays")
+	cmd.Flags().StringSlice(QoSValuesFlag, []string{"1", "1", "1"}, "QoS values: latency, availability, sync scores")
+	cmd.Flags().Uint64(EpochFlag, 0, "Epoch value to be used. If not set, it will be queried from the chain.")
+
+	return cmd
+}
+
+func newRelaySession(
+	specId string,
+	sessionId uint64,
+	cuAmount uint64,
+	providerAddr string,
+	relayNum uint64,
+	qosReport *types.QualityOfServiceReport,
+	epoch int64,
+	chainID string,
+) *types.RelaySession {
+	relaySession := &types.RelaySession{
+		SpecId:      specId,
+		SessionId:   sessionId,
+		CuSum:       cuAmount,
+		Provider:    providerAddr,
+		RelayNum:    relayNum,
+		QosReport:   qosReport,
+		Epoch:       epoch,
+		LavaChainId: chainID,
+	}
+	return relaySession
+}
+
+func extractQoSFlag(qosValues []string) (qosReport *types.QualityOfServiceReport, err error) {
+	if err != nil {
+		return nil, err
+	}
+	// Check if we have exactly 3 values
+	if len(qosValues) != 3 {
+		return nil, utils.LavaFormatError("expected 3 values for QoSValuesFlag", nil, utils.Attribute{Key: "QoSValues", Value: qosValues})
+	}
+	// Convert string values to Dec
+	latency, err := sdk.NewDecFromStr(qosValues[0])
+	if err != nil {
+		return nil, utils.LavaFormatError("invalid latency value", err)
+	}
+
+	availability, err := sdk.NewDecFromStr(qosValues[1])
+	if err != nil {
+		return nil, utils.LavaFormatError("invalid availability value", err)
+	}
+
+	syncScore, err := sdk.NewDecFromStr(qosValues[2])
+	if err != nil {
+		return nil, utils.LavaFormatError("invalid sync score value", err)
+	}
+
+	qosReport = &types.QualityOfServiceReport{
+		Latency:      latency,
+		Availability: availability,
+		Sync:         syncScore,
+	}
+
+	return qosReport, nil
+}
+
+func extractEpoch(clientCtx client.Context, epochValue uint64) (epoch int64, err error) {
+	// If epochValue is not the default value (0 in this case), use it as epoch
+	if epochValue != 0 {
+		return int64(epochValue), nil
+	}
+	status, err := clientCtx.Client.Status(context.Background())
+	if err != nil {
+		return 0, err
+	}
+	latestBlockHeight := status.SyncInfo.LatestBlockHeight
+	fmt.Println("latestBlockHeight: ", latestBlockHeight)
+
+	epoch = calculateEpochFromBlockHeight(latestBlockHeight)
```

Comment:
this is not accurate, you can query the chain for the current epoch using 
lavad q epochstorage show-epoch-details